cmd/build/helmify/static/values.yaml

```diff
@@ -212,8 +212,6 @@ controllerManager:
       #   - ipBlock:
       #       cidr: 0.0.0.0/0
 audit:
-  enablePubsub: false
```

Comment:
I think you removed the wrong value here?

Reply:
yip! I will update that.

Reply:
yea